overlays/agent-modifications-overlay.yaml

```diff
@@ -8,39 +8,35 @@ info:
     before: ""
     type: speakeasy-modifications
 actions:
-  # ===============================================
-  # Agent
-  # ===============================================
-
   - target: $["paths"]["/rest/api/v1/runagent"]["post"]
     update:
-      x-speakeasy-group: client.agents.agent
+      x-speakeasy-group: client.agent
       x-speakeasy-name-override: run
     x-speakeasy-metadata:
```

Comment:
for what it's worth, you can delete all references to `x-speakeasy-metadata` that stuff is purely for the studio